xml/hwbp_registers.xml

```diff
@@ -218,15 +218,13 @@
     </register>
 
     <register name="Hypervisor Context" short="hcontext" address="0x6a8">
-        This optional register is only accessible in S/HS-mode, M-mode and
-        Debug Mode.
+        This optional register may only be implemented if the H extension is
+        implemented. If it is implemented, \RcsrMcontext must also be implemented.
```

Comment:
"may only be implemented" seems ambiguous because "may only" is kind of like "may not" which is like "you may not have a piece of candy."

Somehow this slight tweak seems less ambiguous to me:
```suggestion
        This optional register may be implemented only if the H extension is
        implemented. If it is implemented, \RcsrMcontext must also be implemented.
```

Reply:
Done.